### ISSUE TITLE
Music mod causing crash with null object

### ISSUE THREAD

**Issue by Jamroot** [VazkiiMods/Bliss#99]
[crash-2022-05-31_08.23.56-client.txt](https://github.com/VazkiiMods/Bliss/files/8806477/crash-2022-05-31_08.23.56-client.txt)
[crash-2022-05-31_09.13.29-client.txt](https://github.com/VazkiiMods/Bliss/files/8806478/crash-2022-05-31_09.13.29-client.txt)
[crash-2022-05-31_08.07.06-client.txt](https://github.com/VazkiiMods/Bliss/files/8806480/crash-2022-05-31_08.07.06-client.txt)

Sadly as I'm not too familiar with minecraft's code the most information I can give is what can be gathered by a cursory view of the crash report- however I shall still include it.
Issue appears to be a null variable contained in a map object that is being used by the music controller mod. 

Pack was downloaded from curseforge, Vazkii has helpfully noted the crash appears to be caused by the music mod while unpausing.

**Comment by szunakabeats** [comment]
i think i had kind of the same crash with the music trigger mod
[crash-2022-05-31_17.50.34-client.txt](https://github.com/VazkiiMods/Bliss/files/8806773/crash-2022-05-31_17.50.34-client.txt)

**Comment by gamingoninsulin** [comment]
@FyreAlchemage yes this is the same issue

**Comment by Jamroot** [comment]
the issue appears to be localized to this file https://github.com/TheComputerizer/Music-Triggers/blob/main/src/main/java/mods/thecomputerizer/musictriggers/client/MusicPlayer.java#L439 at the specified line. Could be hotfixed by placing a try catch block and setting variables fadingOut, tempFadeOut, and saveVolOut to some default values in the case of a catch. This wont fix the underlying problem, as far as I can tell there must be some combination of control flow that results in this code block running when volumeLinker has a null value. As far as I can tell, this should only happen if either said control flow occurred at the exact moment the file was run, or this: https://github.com/TheComputerizer/Music-Triggers/blob/main/src/main/java/mods/thecomputerizer/musictriggers/config/ConfigToml.java confusingly named file returned a null value in some of its saved maps, which all appear to be tracking various bits of data.

**Comment by DioEgizio** [comment]
why wontfix?

**Comment by Vazkii** [comment]
It's outside the pack's scope. I'm just waiting on the music triggers update as much as all of you are
